src/queries/fake-class-goal-use.sql

```diff
@@ -0,0 +1,107 @@
+/**
+* This query collects all the fake Monitoring goals used on approved reports within the defined time range.
+*
+* The query results are filterable by the JDI flags. All JDI flags are passed as an array of values
+* The following are the available flags within this script:
+* - jdi.regionIds - one or more values for 1 through 12
+* - jdi.recipients - one or more verbatium recipient names
+* - jdi.grantNumbers - one or more verbatium grant numbers
+* - jdi.goals - one or more verbatium goal text
+* - jdi.status - one or more verbatium statuses
+* - jdi.createdVia - one or more verbatium created via values
+* - jdi.onApprovedAR - true or false
+* - jdi.createdbetween - two dates defining a range for the createdAt to be within
+* - jdi.startDate - two dates defining a range for the startDate to be within
+* - jdi.endDate - two dates defining a range for the endDate to be within
+*
+* zero or more JDI flags can be set within the same transaction as the query is executed.
+* The following is an example of how to set a JDI flag:
+* SELECT SET_CONFIG('jdi.createdbetween','["2023-10-01","2023-10-15"]',TRUE);
+*/
+SELECT
+	gr."regionId",
+	r.name "Recipient Name",
+	gr.number "grant number",
+	ar.reason,
+	ar."startDate",
+	ar."endDate",
+	g.name
+FROM "Goals" g
+JOIN "ActivityReportGoals" arg
+ON g.id = arg."goalId"
+JOIN "ActivityReports" ar
+ON arg."activityReportId" = ar.id
+JOIN "Grants" gr
+ON g."grantId" = gr.id
+JOIN "Recipients" r
+ON gr."recipientId" = r.id
+WHERE ar."calculatedStatus" = 'approved'
+AND g.name like '%CLASS%' 
```

Comment:
Do we want to do an `UPPER(g.name)` here?

Reply:
I think I want to keep case as it is as CLASS is used as a noun using its capitalization. So class and CLASS are different things.